fossunited/chapters/web_template/foss_club___support_from_foss_united/foss_club___support_from_foss_united.html

```diff
@@ -98,11 +98,11 @@ <h2>Support from FOSS United</h2>
         {% if support_title %}
           <div class="support-card">
             <img src="{{ support_icon }}" alt="" />
-            <h5>{{ support_title }}</h5>
+            <h3>{{ support_title }}</h3>
             <p>{{ support_text }}</p>
           </div>
         {% endif %}
       {% endfor %}
     </div>
   </div>
-</div>
+</div>
```

Comment:
```suggestion
</div>

```

re-add empty line at EOF (end of file)